package.json

```diff
@@ -64,15 +64,15 @@
   "dependencies": {
     "appium-adb": "^9.13.1",
     "appium-android-driver": "^5.14.0-alpha2",
-    "appium-chromedriver": "^5.3.1",
+    "appium-chromedriver": "^5.5.1",
     "appium-uiautomator2-server": "^5.12.2",
     "asyncbox": "^2.3.1",
     "axios": "^1.x",
     "bluebird": "^3.5.1",
     "css-selector-parser": "^2.2.3",
     "lodash": "^4.17.4",
     "portscanner": "^2.2.0",
-    "source-map-support": "^0.x",
+    "source-map-support": "^0.5.21",
```

Comment:
by doing this we only allow minor version updates. Do we want that? Also, in the module readme I've seen it is not needed anymore after Node 12, so can we simply get rid of it?